spyder/plugins/explorer/widgets.py

```diff
@@ -192,6 +193,41 @@ def icon(self, icontype_or_qfileinfo):
         return icon
 
 
+class ColorModel(QFileSystemModel):
+    """FileSystemModel providing a color-code for different commit-status."""
+    def __init__(self, *args, **kwargs):
+        self.vcs_state = []
+        normalstyle = CONF.get('appearance', 'selected') + '/normal'
+        self.color_array = [QColor(CONF.get('vcs', 'color/untracked')),
+                            QColor(CONF.get('vcs', 'color/ignored')),
+                            QColor(CONF.get('vcs', 'color/modified')),
+                            QColor(CONF.get('vcs', 'color/added')),
+                            QColor(CONF.get('appearance', normalstyle)[0])]
+        self.root_path = ''
+        super(ColorModel, self).__init__(*args, **kwargs)
+
+    def setVCSState(self, root_path):

```

Comment:
Please use standard Pythonic ``snake_case()`` for your function and method names unless you are overriding a Qt method.